examples/mobile/UIComponents/components/view-controls/sub-tab/one.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js"></script>
+	<title>SubTab One</title>
+</head>
+
+<body>
+	<div class="ui-page">
+		<header>
+			<div class="ui-appbar-title-container">
+				<span class="ui-appbar-title">Sample tab</span>
+			</div>
+		</header>
+		<div class="ui-tabs">
+			<div class="ui-sub-tab ui-sub-tab-static">
+				<ul>
+					<li class="ui-li-anchor">
+						<a class="ui-tab-active" href="#">
+							One
+						</a>
+					</li>
+					<li class="ui-li-anchor">
+						<a href="#">
+							Two
+						</a>
+					</li>
```

Comment:
This example has two subtabs  - similar to sub-tab-2-items.html , what is teh sense of it?

Reply:
You mean `two.html`, yes? This page looks the same because sample in guideline looks the same for all pages, but in real apps these file will be different.
I'm not sure what is better and clear.
Instead of this file, I would have to make change in `sub-tab-2-items.html` file. It will be strange why tab `two` links to page `one.html` 
```
<li class="ui-li-anchor">
	<a class="ui-tab-active" href="./one.html">
		One
	</a>
</li>
<li class="ui-li-anchor">
	<a href="./**one**.html">
		Two
	</a>
</li>
```
I mean `./one.html` as link for item `two`.